WinAppDriver UI Tests (#89)
microsoft/Xaml-Controls-Gallery: UITests/README.md

> # UWPControls

Reviewer: ```suggestion
# UI Testing with WinAppDriver
```

> UWPControls is a collection of test scenarios that covers basic interactions with [Universal Windows Platform controls (UI elements)](https://docs.microsoft.com/en-us/windows/uwp/controls-and-patterns/). Use this as a reference on how to interact with certain UWP controls you have in your application.

Reviewer: I don't understand this paragraph. What is "UWPControls"? Does that refer to the Xaml Controls Gallery? I don't think we need this paragraph if so.

> The test scenarios are written against [AppUIBasics](../../ApplicationUnderTests/AppUIBasics) application that contains all UWP basic controls. This application needs to be installed once in the machine you are running your test. Follow the instruction on the application [README](../../ApplicationUnderTests/AppUIBasics/README.md).

Reviewer: As above, what is "AppUIBasics"? Does it reference the Xaml Controls Gallery?

Maybe both of these two paragraphs can be condensed into something like:

```suggestion
The test scenarios are written to test the controls in the Xaml Controls Gallery app, which contains all UWP Xaml controls. The Xaml Controls Gallery app needs to be installed on the machine where you are running your tests. Follow the instructions in the root README to install the app.
```

Reviewer: Reactivating since the feedback hasn't been addressed.

## Requirements

- Windows 10 PC with the latest Windows 10 version (Version 1607 or later)
- Microsoft Visual Studio 2015 or later

> - [AppUIBasics](../../ApplicationUnderTests/AppUIBasics) application

Reviewer: ```suggestion

```

## Getting Started

> 1. Open `UWPControls.sln` in Visual Studio

Reviewer: Is this accurate? Or should it be "UITest.sln"?
